src/screens/NewsScreen/index.js

```diff
@@ -226,7 +226,7 @@ const NewsScreen = () => {
   const keyExtractor = React.useCallback((item, index) => index.toString(), []);
 
   return (
-    <SafeAreaProvider style={styles.container}>
+    <View style={styles.container}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `SafeAreaProvider` to `View` might cause layout issues on devices with notches or rounded corners. The `SafeAreaProvider` component provides padding to avoid the notch and home indicator on modern iOS devices, as well as the status bar and navigation bar on Android devices. If you're sure this change won't affect your layout, ignore this comment. Otherwise, consider reverting back to using `SafeAreaProvider`.

```diff
- <View style={styles.container}>
+ <SafeAreaProvider style={styles.container}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->